scripts/testdata/satellite-config.yaml.lock

```diff
@@ -89,7 +89,7 @@ compensation.withheld-percents: 75,75,75,50,50,50,25,25,25,0,0,0,0,0,0
 # console.frame-ancestors: tardigrade.io
 
 # url link for gateway credentials requests
-# console.gateway-credentials-request-url: ""
+# console.gateway-credentials-request-url: "https://auth.tardigradeshare.io"
```

Comment:
Why do we keep the configuration value when is it the same as the default?

Reply:
I don't understand that question. Currently we don't have a default value and this PR will set one. If I don't update the config lock file a unit test will fail. I always have to update the default value and the config lock file.

Reply:
Sorry, I forgot that we have to update the config lock file with the default value. I got confused with another of your PRs where a default value was removed from some configuration file because we only want to keep it in the configuration file if the values have to be set to a different value than the default one.